docs/reference-guide/streams/overview.md

```diff
@@ -56,15 +57,39 @@ Drops a stream with name `<stream name>`.
 ## Start a stream
 
 ```cypher
-START STREAM <stream name> [LIMIT <count> BATCHES];
+START STREAM <stream name>;
 START ALL STREAMS;
 ```
 Starts a stream (or all streams) with name `<stream name>`.
 
-`<count>` denotes the total number of processed batches.
-
-When a stream is started, it should resume from the last committed
-offset.
+When a stream is started, it should resume from the last committed offset. If
+there is no committed offset for the consumer group, then the largest offset
+will be used, therefore only the new messages will be consumed.
+
+### At least once semantics
+
+In stream processing it is important to have some guarantees about how failures
+are handled. In case connecting external application such as Memgraph to Kafka
+streams, there are two possible ways to handle failures during message
```

Comment:
```suggestion
 there are two possible ways to handle failures during message
```

Reply:
I did your proposed changes but wrapped the lines differently to remain into the 80 char limit.